cell_classification/configs/params.toml

```diff
@@ -26,5 +26,12 @@ contrast_prob = 0.5
 contrast_min = 0.8
 contrast_max = 1.2
 batch_size = 4
+steps_per_epoch = 100
+loss_fn = "BinaryCrossentropy"
+loss_selective_masking = true
+[loss_kwargs]
+from_logits = true
+label_smoothing = 0.1
```

Comment:
It doesn't look like label_smoothing is ever popped from the params. Is this handled automatically somewhere?

Reply:
I assume that with only two classes, this won't have a huge impact, but given how easy it is to implement probably worth including?

Reply:
Not sure if I understand this correctly. So under `[loss_kwargs]` you can put all arguments that you want to put into the init of your tf.keras.loss. In [model_builder.py](https://github.com/angelolab/cell_classification/blob/8b7816cbb8fe041c34c40402d9949e73f6705159/cell_classification/model_builder.py#L150)  we pop `loss_kwargs` from params and it is used by the keras loss function.